requirements.in

```diff
@@ -1,4 +1,4 @@
-https://github.com/codecov/shared/archive/b2798abaa2b47d8834cf7d552cc892efbff01b70.tar.gz#egg=shared
+https://github.com/codecov/shared/archive/a595bf868c4486fffc187bc930dd59a928a93dd0.tar.gz#egg=shared
```

Comment:
did you `pip-compile requirements.in`? the shared sha isn't updated in requirements.txt

Reply:
Mmmm, I should have, but this is still pending on this https://github.com/codecov/shared/pull/178 being reviewed, so I'll update the SHA after anyway and will rerun to make sure. Good eyes though! :)